test/helpers/contract/bootstrap.js

```diff
@@ -1,12 +1,18 @@
 const Ganache = require(process.env.TEST_BUILD
   ? "../build/ganache.core." + process.env.TEST_BUILD + ".js"
-  : "../../index.js");
+  : "../../../index.js");
 const Web3 = require("web3");
-
 const { join } = require("path");
 const { compileAndDeploy } = require("./compileAndDeploy");
 
-const setUp = (mainContractName = "", subContractNames = [], options = {}, contractPath = "../contracts/") => {
+/**
+ * @param {string} mainContractName Main contract filename (withouth file extension)
+ * @param {string|Array} subContractNames Array of supporting contract filenames (without file extension)
+ * @param {Object} options Provider options
+ * @param {string} contractPath Relative path to contract
```

Comment:
Relative to what? This module, or the caller? Will absolute work?

Reply:
It's relative to the module. It's defaults to the contract directory where all contracts files will eventually live.